.github/workflows/nightly-ceremony.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: dev
+          ref: v1.2.1
```

Comment:
Can we use wildcard here or we don't need to support previous and future versions?

Reply:
so these ceremony zkeys only work with 1.2.0 and 1.2.1, now with dev we have changes to the circuits and require a new ceremony. But we should probably get an audit first. 
And the problem here is that unless we publish a 1.2.2 (starting from 1.2.1 and not current dev) with the ceremony test fix we won't be able to run successfully these tests 🤔